src/Microsoft.DotNet.Interactive.Http/VariableRouter.cs

```diff
@@ -62,20 +66,26 @@ private async Task BatchVariableRequest(RouteContext context)
                     var propertyBag = new JObject();
                     response[kernelName] = propertyBag;
                     var targetKernel = GetKernel(kernelName);
-                    if (targetKernel is DotNetKernel languageKernel)
+                    if (targetKernel is null)
+                    {
+                        context.Handler = async httpContext =>
+                        {
+                            httpContext.Response.StatusCode = 400;
+                            await httpContext.Response.WriteAsync($"kernel {kernelName} not found");
+                            await httpContext.Response.CompleteAsync();
+                        };
+                        return;
+                    }
+                    
+                    if (targetKernel.SupportsCommand<RequestValue>() || targetKernel is DotNetKernel)
```

Comment:
I do still think we should get rid of `DotNetKernel`.

Reply:
done